MyCryptoHQ/MyCrypto
# Add additional validation to Amount input field

Closes #1789

### Description
Users were able to enter a value with multiple decimal separators without being shown an error.

### Changes

* The `validDecimal` validator now checks for only a single decimal.
* `AmountField`s will now show an error after being touched if there is no value.

### Steps to Test

1. Navigate to the "Request Payment" tab in "Account".
2. Enter a value with multiple decimals (such as "1.2.3").
3. The field should become red after blurring, indicating an error.

### Screenshots
<img width="850" alt="screen shot 2018-06-22 at 5 16 19 pm" src="https://user-images.githubusercontent.com/10479826/41801412-04b370fa-7640-11e8-972d-2c87de630f27.png">
